### ISSUE TITLE
Fundamental styles missing without ftw.theming

### ISSUE THREAD

**Issue by djowett-ftw** [4teamwork/ftw.simplelayout#584]
e.g. 

<img width="903" alt="Screenshot 2020-01-06 at 16 26 36" src="https://user-images.githubusercontent.com/54800639/71974668-57ad5980-3212-11ea-9a5b-f4d9843aca70.png">

as discussed yesterday, should we just depend on ftw.theming (for now at least)?

**Comment by jone** [comment]
I think we should depend on `ftw.theming`, because:
1) We are not maintaing a non-`ftw.theming`-setup (we had that in the beginning but in practice we cannot ensure quality for two sets of CSS, as the current state shows).
2) `ftw.theming` is not destructive. That means if you don't install a `ftw.theming`-based theme, it should not result in more css other than the CSS of the installed add-ons such as `ftw.simplelayout`.

@maethu what is your opinion on that?

**Comment by djowett-ftw** [comment]
> I think we should depend on `ftw.theming`

Agreed - it's more straightforward for both us - and potential adopters.  (Not to exclude PRs from anyone wanting to maintain a non `ftw.theming` theme, but there has been no input like that I'm aware of)
